apps/deploy-web/src/services/app-di-container/app-di-container.ts

```diff
@@ -63,12 +63,7 @@ export const createAppRootContainer = (config: ServicesConfig) => {
       container.applyAxiosInterceptors(new AuthHttpService(apiConfig), {
         request: [container.authService.withAnonymousUserHeader]
       }),
-    providerProxy: () =>
-      new ProviderProxyService(
-        container.applyAxiosInterceptors(container.createAxios({ baseURL: config.BASE_PROVIDER_PROXY_URL }), {
-          request: [container.authService.withAnonymousUserHeader]
-        })
-      ),
+    providerProxy: () => new ProviderProxyService(container.createAxios({ baseURL: config.BASE_PROVIDER_PROXY_URL })),
```

Comment:
issue: provider-proxy requests won’t contain trace parent. You need to remove interceptor from the list but keep everything elser